bitnami/kubeapps/crds/apprepository-crd.yaml

```diff
@@ -10,4 +10,96 @@ spec:
     plural: apprepositories
     shortNames:
       - apprepos
-  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      storage: true
+      served: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required: [ "spec" ]
```

Comment:
Is it the typical syntax for the `required` ? Or should we use the yaml way? I mean, I'm ok with both approaches, just trying to make it similar as in other Bitnami's CRD.

``` 
required:  
  - blabla
```

Reply:
i think i have written it that way from doing that way before, and also as it is how it is in the kubernetes doc (but the doc might try to keep samples small).
I will update to be more consistent